testsuite/sclang/TestLexerTargetedBrutal.sc

```diff
@@ -14,7 +14,7 @@ TestLexerTargetedBrutal : AbstractBrutalTest {
 		alphanumAlphabet = alphanumAlphabet.collect(_.asString);
 		alphanumAlphabet = alphanumAlphabet ++ ["pi", "+", "-", "."];
 
-		alphabets = [
+		alphabets = Dictionary[
```

Comment:
Fair; speed isn't a concern here though, and I figured using an IdentityDict might imply something more clever is happening than is really the case. :) Do you mind if I leave it?